hera_mc/tests/test_node.py

```diff
@@ -33,395 +31,414 @@
 
 node_power_example_dict = {
```

Comment:
I would echo the same apprehension here about defining these more or less globally and recommend fixtures

Reply:
fixed.